src/wms/src/test/resources/org/geoserver/wms/caps130.xml

```diff
@@ -1,86 +1,96 @@
 <WMS_Capabilities version="1.3.0" updateSequence="1159"
-  xmlns="http://www.opengis.net/wms" xmlns:xlink="http://www.w3.org/1999/xlink"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://www.opengis.net/wms http://schemas.opengis.net/wms/1.3.0/capabilities_1_3_0.xsd">
-  <Service>
-    <Name>WMS</Name>
-    <Title>WMS</Title>
-    <Abstract>Minimal test caps document for a WMS 1.3 server</Abstract>
-    <KeywordList>
-      <Keyword>GEOSERVER</Keyword>
-    </KeywordList>
-    <OnlineResource xlink:type="simple" xlink:href="http://mock.test.geoserver.org" />
-  </Service>
-  <Capability>
-    <Request>
-      <GetCapabilities>
-        <Format>text/xml</Format>
-        <DCPType>
-          <HTTP>
-            <Get>
-              <OnlineResource xlink:type="simple"
-                xlink:href="http://mock.test.geoserver.org/wms13?SERVICE=WMS&amp;" />
-            </Get>
-          </HTTP>
-        </DCPType>
-      </GetCapabilities>
-      <GetMap>
-        <Format>image/png</Format>
-        <DCPType>
-          <HTTP>
-            <Get>
-              <OnlineResource xlink:type="simple"
-                xlink:href="http://mock.test.geoserver.org/wms13?SERVICE=WMS&amp;" />
-            </Get>
-          </HTTP>
-        </DCPType>
-      </GetMap>
-      <GetFeatureInfo>
-        <Format>text/plain</Format>
-        <Format>application/vnd.ogc.gml</Format>
-        <Format>application/vnd.ogc.gml/3.1.1</Format>
-        <Format>text/html</Format>
-        <DCPType>
-          <HTTP>
-            <Get>
-              <OnlineResource xlink:type="simple"
-                xlink:href="http://mock.test.geoserver.org/wms13?SERVICE=WMS&amp;" />
-            </Get>
-          </HTTP>
-        </DCPType>
-      </GetFeatureInfo>
-    </Request>
-    <Exception>
-      <Format>XML</Format>
-      <Format>INIMAGE</Format>
-      <Format>BLANK</Format>
-    </Exception>
-    <Layer>
-      <Title>Root</Title>
-      <!--Limited list of EPSG projections: -->
-      <CRS>EPSG:4326</CRS>
-      <CRS>CRS:84</CRS>
-      <EX_GeographicBoundingBox>
-        <westBoundLongitude>-180</westBoundLongitude>
-        <eastBoundLongitude>180</eastBoundLongitude>
-        <southBoundLatitude>-90</southBoundLatitude>
-        <northBoundLatitude>90</northBoundLatitude>
-      </EX_GeographicBoundingBox>
-      <Layer queryable="1">
-        <Name>world4326</Name>
-        <Title>world4326</Title>
-        <CRS>EPSG:4326</CRS>
-        <CRS>CRS:84</CRS>
-        <EX_GeographicBoundingBox>
-          <westBoundLongitude>-180</westBoundLongitude>
-          <eastBoundLongitude>180</eastBoundLongitude>
-          <southBoundLatitude>-90</southBoundLatitude>
-          <northBoundLatitude>90</northBoundLatitude>
-        </EX_GeographicBoundingBox>
-        <BoundingBox CRS="EPSG:4326" minx="-90" miny="-180"
-          maxx="90" maxy="180" />
-      </Layer>
-    </Layer>
-  </Capability>
+	xmlns="http://www.opengis.net/wms"
```

Comment:
What's this change, unrelated format changes? If yes please remove them.

Reply:
looks like my eclipse is up to no good, I will fix this